.circleci/config.yml

```diff
@@ -7,10 +7,10 @@ parameters:
     default: syscon-alerts-security
   test-executor-packages:
     type: string
-    default: "activities alerts appointments courtsentencing csip incidents locations sentencing visits personalrelationships casenotes organisations"
+    default: "activities alerts appointments courtsentencing csip incidents locations sentencing visits personalrelationships casenotes organisations coreperson visitbalances"
```

Comment:
one of the parallel tests was taking 7 minutes, turns out that each time a new migration is added they need to be added here so that can still be split